app/controllers/Api.scala

```diff
@@ -33,15 +33,15 @@ class Api (override val stores: DataStores,
 
   def getMediaAtom(id: String) = APIAuthAction { implicit req =>
     previewDataStore.getAtom(id) match {
-      case Some(atom) => Ok(Json.toJson(atom))
-      case None => NotFound(jsonError(s"no atom with id $id found"))
+      case Right(atom) => Ok(Json.toJson(atom))
+      case _ => NotFound(jsonError(s"no atom with id $id found"))
```

Comment:
Definitely, I was unsure what to return. Probably an InternalServerError like Api2 I guess?